src/code.google.com/p/codesearch/index/mmap_bsd.go

```diff
@@ -12,6 +12,17 @@ import (
   "syscall"
 )
 
+// An mmapData is mmap'ed read-only data from a file.
```

Comment:
We might be able to move the _bsd and _linux code into a single mmap_unix.go file.